model_metadata.py

```diff
@@ -0,0 +1,61 @@
+import os
+import re
+import json
+import yaml
+from dotenv import load_dotenv
+from azure.ai.ml import MLClient, Input
+from azure.ai.ml.entities import Model
+from azure.ai.ml.constants import AssetTypes
+from azure.identity import DefaultAzureCredential
+
+load_dotenv()
+
+NACHET_SUBSCRIPTION_ID = os.getenv("NACHET_SUBSCRIPTION_ID")
+NACHET_RESOURCE_GROUP = os.getenv("NACHET_RESOURCE_GROUP")
+NACHET_WORKSPACE = os.getenv("NACHET_WORKSPACE")
+
+def generate_model_metadata():
+    """
+    Retrieves deployed online_endpoints and generates metadata json 
+    """
+
+    model_metadata = []
+    model_json = {}
+
+    ml_client = MLClient(DefaultAzureCredential(), NACHET_SUBSCRIPTION_ID, NACHET_RESOURCE_GROUP, NACHET_WORKSPACE)
```

Comment:
reformat your extra long lines to improve readability (parenthesis allow you to continue multilines)

```
ml_client = MLClient(
    DefaultAzureCredential(), 
    NACHET_SUBSCRIPTION_ID,
    NACHET_RESOURCE_GROUP,
    NACHET_WORKSPACE)
```

Reply:
Done